subgraphs/_reference_/src/sdk/protocols/bridge/pool.ts

```diff
@@ -24,13 +24,17 @@ import {
   RewardTokenType,
 } from "../../util/constants";
 import { exponentToBigDecimal } from "../../util/numbers";
-import { sortArrayByReference, sortBytesArray, updateArrayAtIndex } from "../../util/arrays";
+import {
+  sortArrayByReference,
+  sortBytesArray,
+  updateArrayAtIndex,
+} from "../../util/arrays";
 import { TokenManager } from "./tokens";
 import { PoolSnapshot } from "./poolSnapshot";
 import { SDK } from ".";
 
 type onCreatePoolCallback<T> = (
-  event: ethereum.Event,
+  event: ethereum.Block,
```

Comment:
The callback might need to use info from the event. Specially because AssemblyScript doesn’t allow closures. Can you make it `CustomEventType` instead so we don’t remove context?

Reply:
makes a lot of sense, updated!